Explorer/Assets/DCL/Chat/Assets/Chat.prefab

```diff
@@ -215,7 +215,7 @@ MonoBehaviour:
   m_TargetGraphic: {fileID: 2680055985081135439}
   m_HandleRect: {fileID: 7426884741399343867}
   m_Direction: 2
-  m_Value: 0
```

Comment:
damn, I have a refinement branch that changed a lot the chat prefab as well.....